frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -21,7 +22,7 @@ local ReaderWikipedia = ReaderDictionary:extend{
     -- identify itself
     is_wiki = true,
     wiki_languages = {},
-    no_page = _("No wiki page found."),
+    no_page = _("No wikipedia page found."),
```

Comment:
How about something like `There were no Wikipedia articles matching the search term.`